source/gui/addonStoreGui/controls/messageDialogs.py

```diff
@@ -612,7 +612,7 @@ def delayCreateDialog():
 				).start()
 
 			case _:
-				raise NotImplementedError("Unknown automatic update setting")
+				raise NotImplementedError("Unknown add-ons update setting")
```

Comment:
I think this is a bit misleading - as it doesn't affect the update behaviour from the update panel in the add-on store